client/src/components/Authorization/Login.js

```diff
@@ -192,20 +193,20 @@ const Login = props => {
                     </Link>
                   </div>
 
-                  <button
+                  <Button
                     id="cy-login-submit"
                     type="submit"
                     className="btn-submit"
                     disabled={isSubmitting}
+                    color="colorPrimary"
                   >
                     {isSubmitting ? "Please wait..." : "Login"}
-                  </button>
-
+                  </Button>
                   {/* <button className="btn-without-saving"> */}
                   <Link to="/calculation/1">
-                    <button className="btn-without-saving">
+                    <Button className="btn-without-saving">
```

Comment:
should get rid of css classname and use styles from the Button component props to determine the style